src/pages/NewResearcherConsole.js

```diff
@@ -131,6 +135,11 @@ export default function NewResearcherConsole(props) {
     }
   };
 
+  const resumeDARApplication = (darCollection) => {
+    const referenceId = darCollection.referenceId || flow(get('dars'), keys, head)(darCollection);
```

Comment:
I originally used `flow(get('dars'), keys, head)(darCollection)` so that the review button could go to `/dar_application/{referenceId}` before the `/dar_application_review/{collectionId}` page existed so the button would no longer throw up an error page. Since the resume button is only visible for drafts, which from my understanding always have `referenceId`s and never have a `dars` property in this table, I don't think `flow(get('dars'), keys, head)(darCollection)` is ever reached or useful.